front/app/containers/IdeasShow/components/ProposalInfo/index.tsx

```diff
@@ -25,6 +25,7 @@ const componentMap = {
   answered: Answered,
   threshold_reached: ThresholdReached,
   ineligible: Ineligible,
+  custom: () => <></>,
 };
```

Comment:
@IvaKop I just haven't really seen this format before - is this how we want to conditionally render components in React? :slightly_smiling_face: